src/hooks/useDelegation.ts

```diff
@@ -4,13 +4,13 @@ import useAsyncMemo from "decentraland-gatsby/dist/hooks/useAsyncMemo";
 const ENDPOINT = `https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot`
 const QUERY = `
 query ($space: String!, $address: String!) {
-  delegatedTo: delegations(where: { space: $space, delegator: $address }, first: 1, orderBy: timestamp, orderDirection: desc) {
+  delegatedTo: delegations(where: { space_in: ["", $space], delegator: $address }, first: 1, orderBy: timestamp, orderDirection: desc) {
```

Comment:
What happen if we get both delegations? shouldn't we handle the case where we get 2 responses?

Reply:
looks good to me